superset/sql_parse.py

```diff
@@ -41,7 +50,6 @@
 CTE_PREFIX = "CTE__"
 logger = logging.getLogger(__name__)
 
-
```

Comment:
nit: maybe add this back to avoid making unrelated changes

Reply:
Done.

Reply:
removed line seems to still be here